### ISSUE TITLE
Generate g higher resolution images

### ISSUE THREAD

**Issue by yangyvhang666** [tohinz/ConSinGAN#9]
Hi,I find that if i modify the resolution of the generated image,the generated images get bad results.Should I increase the number of  concurrently trainde stages?

**Comment by tohinz** [comment]
Hi, I suggest increasing the number of `--trained_stages` (default is 6) in general. This should already improve results for higher resolutions.
If this is not enough you can also increase the `--lr_scale` (default is 0.1 - you could try 0.5).
Finally, it might be necessary to train for more iterations on higher stages/resolutions and/or adapt other hyperparameters (receptive field of the discriminator/generator, learning rate, etc).

**Comment by yangyvhang666** [comment]
Thanks for your reply,I tried increase the --trained_stages and the lr_scale,but it might did't work,I want to know the --max_size and the --min_size should be setted to have some kind of proportional relationship?

**Comment by tohinz** [comment]
`--min_size` should be left at some small value, so I wouldn't change it or at least not increase it. `--min_size` and `--max_size` do not need to have a proportional relationship, rather `--max_size` and `--trained_stages` should be somewhat proportional, i.e. the larger `--max_size` the larger `--trained_stages` should be.
How high did you set `--trained_stages` to be? What exactly is the problem with the higher-resolution images? Is the global structure wrong or are there artifacts in the images?
